docker-compose.yml

```
@@ -26,6 +27,7 @@ services:
     # user: ${MYUID:-0}
     build:
       context: .
+    init: true
```

I believe this init should go below `cache_from`. The build is failing to parse the yaml file and is throwing an error.

```suggestion
    build:
      context: .
      cache_from:
        - docker.sendgrid.net/sendgrid/ui-components:latest
    init: true
```